cli/debug.go

```diff
@@ -119,9 +124,73 @@ func runDebugRangeDescriptors(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var debugRaftLogCmd = &cobra.Command{
+	Use:   "raft-log [directory] [range id]",
+	Short: "print the raft log for a range",
+	Long: `
+  Prints all log entries in a store for the given range.
```

Comment:
`Long` help text is not supposed to be indented. Looks like this is done incorrectly in `debug.go`. Mind fixing while you're here?